README.md

```diff
@@ -135,10 +166,86 @@ This workflow shows how to spawn one job per Gazebo release. It iterates over al
             fi
 ```
 
+- *Using Ubuntu docker containers*
+
+```yaml
+  jobs:
+    test_gazebo:
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.docker_image }}
+    strategy:
+      fail-fast: false
+      matrix:
+        gazebo_distribution:
+          - citadel
+          - fortress
+          - garden
+          - harmonic
+        include:
+          # Gazebo Citadel (Dec 2019 - Dec 2024)
+          - docker_image: ubuntu:focal
+            gazebo_distribution: citadel
+
+          # Gazebo Fortress (Sep 2021 - Sep 2026)
+          - docker_image: ubuntu:focal
+            gazebo_distribution: fortress
+
+          # Gazebo Garden (Sep 2022 - Nov 2024)
+          - docker_image: ubuntu:focal
+            gazebo_distribution: garden
+
+          # Gazebo Harmonic (Sep 2023 - Sep 2028)
+          - docker_image: ubuntu:jammy
+            gazebo_distribution: harmonic
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4.0.3
+        with:
+          node-version: '20.x'
+      - name: 'Check Gazebo installation on Ubuntu runner'
+        uses: ./
+        with:
+          required-gazebo-distributions: ${{ matrix.gazebo_distribution }}
+      - name: 'Test Gazebo installation'
+        run: |
+          if command -v ign > /dev/null; then
+            ign gazebo --versions
+          elif command -v gz > /dev/null; then
+            gz sim --versions
+          else
+            echo "Neither ign nor gz command found"
+            exit 1
+          fi
+```
+
 #### Using pre-release and/or nightly Gazebo binaries
 
 This workflow shows how to use binaries from [pre-release] or [nightly] Gazebo repositories instead of the stable repository by setting the `use-gazebo-prerelease` or `use-gazebo-nightly` to `true`.
 
+- *Using GitHub-hosted runners*
+
+```yaml
+  jobs:
+    test_gazebo:
+        runs-on: ubuntu-24.04
+        steps:
+          - uses: actions/checkout@v4
+          - uses: actions/setup-node@v4.0.2
+            with:
+              node-version: '20.x'
+          - name: 'Check Gazebo installation on Ubuntu runner'
+            uses: gazebo-tooling/setup-gazebo@<full_commit_hash>
+            with:
+              required-gazebo-distributions: 'harmonic'
+              use-gazebo-prerelease: 'true'
+              use-gazebo-nightly: 'true'
+          - name: 'Test Gazebo installation'
+            run: 'gz sim --versions'
+```
+
+- *Using Ubuntu docker containers*
```

Comment:
I think that we can remove the option of the docker containers for prerelease and nightlies in favour of reducing the README lenght.